mathematica/mathematica_test.cpp

```diff
@@ -116,7 +118,12 @@ TEST_F(MathematicaTest, ToMathematica) {
     EXPECT_EQ("Minus[0]", ToMathematica(-0.0));  // Not that this does anything.
     EXPECT_EQ("Infinity", ToMathematica(Infinity<double>));
     EXPECT_EQ("Minus[Infinity]", ToMathematica(-Infinity<double>));
-    EXPECT_EQ("Minus[Indeterminate]", ToMathematica(Sqrt(-1)));
+    EXPECT_EQ("Indeterminate", ToMathematica(NaN<double>));
+    EXPECT_EQ("Minus[Indeterminate]", ToMathematica(-NaN<double>));
+
+    // The sign of √-1 varies between Clang versions.
```

Comment:
I am somewhat puzzled: it should just call the intrinsic (`PRINCIPIA_USE_SSE3_INTRINSICS` is false in debug on Windows, but I don’t think we have a debug build on *nix).
Is this a Rosetta2 thing? Or is it some chip that has opinions on the sign of i?
https://github.com/mockingbirdnest/Principia/blob/88817a719d78efdfac1435b7e1ad6be1e3e43155/quantities/elementary_functions_body.hpp#L75-L81

Reply:
It looks like this was an x86 vs ARM difference which I mistakenly thought was a clang difference. I reverted it.